cmd/mt-parrot/generate.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"fmt"
+	"github.com/grafana/metrictank/clock"
+	"github.com/grafana/metrictank/schema"
+	log "github.com/sirupsen/logrus"
+	"sync/atomic"
+)
+
+func produceTestMetrics(schemas []*schema.MetricData) {
```

Comment:
why is this variable called schemas? should probably be "metrics" ?

Reply:
We're constantly overwriting the timestamp/value.
My thought was that `schema` made it clear that this is a reusable item and not a single metric datum

Reply:
b5771cbe7a9cfe7ba71740015b93ef7ee06bd05f